lib/auth/auth.go

```diff
@@ -2016,7 +2017,8 @@ type AppTestCertRequest struct {
 // GenerateUserAppTestCert generates an application specific certificate, used
 // internally for tests.
 func (a *Server) GenerateUserAppTestCert(req AppTestCertRequest) ([]byte, error) {
-	userState, err := a.GetUserOrLoginState(context.Background(), req.Username)
+	ctx := context.Background()
```

Comment:
Ya, this one is debatable. I think `context.Background()` is more appropriate at the moment since there is no plan to propagate a proper context here.

Reply:
Sure - I tend to reserve context.Background() for very intentional uses - but I'm not overly bothered by it's usage here.